HungryforApples/app/src/main/java/com/exeinformatique/hungryforapples/MainActivity.java

```diff
@@ -80,5 +88,22 @@ private void generateMarkers() {
             }
         }
         return position;
+
+        setListeners();
+    }
+
+    public void setListeners(){
+        findViewById(R.id.btn_gotoViewRestaurants).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                gotoViewRestaurants();
+            }
+        });
+    }
+
+    private void gotoViewRestaurants(){
+        Intent gotoViewRestaurant = new Intent(this, ViewRestaurantsActivity.class);
+        startActivity(gotoViewRestaurant);
```

Comment:
_startActivity(new Intent(this, ViewRestaurantsActivity.class));_
Parce que pourquoi pas lol